tendrl/ceph_integration/manager/pool_request_factory.py

```diff
@@ -189,8 +189,24 @@ def update(self, pool_id, attributes):
             )
 
     def create(self, attributes):
-        commands = [('osd pool create', {'pool': attributes['name'],
-                                         'pg_num': attributes['pg_num']})]
+        if 'type' in attributes and attributes['type'] == "erasure":
+            commands = [(
+                'osd pool create',
+                {
+                    'pool': attributes['name'],
+                    'pg_num': attributes['pg_num'],
+                    'pool_type': attributes['type'],
```

Comment:
We are based out of old calamari so had to pass the details like this. I am not sure how we would be able to utilize newer calamari. At least old calamari code I checked and type, erasure_code_profile were being sent like this.